vmm/src/default_syscalls.rs

```diff
@@ -9,47 +9,32 @@ use seccomp::{
 
 /// List of allowed syscalls, necessary for Firecracker to function correctly.
 pub const ALLOWED_SYSCALLS: &[i64] = &[
-    libc::SYS_read,
-    libc::SYS_write,
-    libc::SYS_open,
+    libc::SYS_accept,
+    libc::SYS_bind,
     libc::SYS_close,
-    libc::SYS_stat,
+    libc::SYS_dup,
+    libc::SYS_epoll_ctl,
+    libc::SYS_epoll_pwait,
+    libc::SYS_exit,
+    libc::SYS_exit_group,
+    libc::SYS_fcntl,
     libc::SYS_fstat,
```

Comment:
`strace` can't help with stack traces when jailed because it uses /proc/<pid>/smaps to decode the call stacks, and with the jailer's new pid namespace, it doesn't work anymore. I'm pretty sure it originates in the `fs::metadata` call.